src/main/java/uk/gov/justice/laa/crime/cfecrime/utils/RequestHandler.java

```diff
@@ -2,44 +2,98 @@
 
 import lombok.experimental.UtilityClass;
 import lombok.extern.slf4j.Slf4j;
-import uk.gov.justice.laa.crime.cfecrime.api.Result.Outcome;
+import uk.gov.justice.laa.crime.cfecrime.Exceptions.UndefinedOutcomeException;
 import uk.gov.justice.laa.crime.cfecrime.api.*;
+import uk.gov.justice.laa.crime.cfecrime.api.stateless.StatelessApiRequest;
+import uk.gov.justice.laa.crime.cfecrime.api.stateless.StatelessApiResponse;
+import uk.gov.justice.laa.crime.cfecrime.cma.stubs.LocalCmaService;
+import uk.gov.justice.laa.crime.cfecrime.enums.Outcome;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.CaseType;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.FullAssessmentResult;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.InitAssessmentResult;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.MagCourtOutcome;
 
 @Slf4j
 @UtilityClass
 public class RequestHandler {
+    private LocalCmaService cmaService = new LocalCmaService();
+    private StatelessApiResponse statelessApiResponse = null;
+    private StatelessApiRequest cmaRequest = new StatelessApiRequest();
+    private CaseType caseType =  null;
+    private MagCourtOutcome magCourtOutcome = null;
```

Comment:
This pattern isn't thread-safe. We should always pass variables directly to methods that need them, not use side-effects as we have been badly bitten by them in cfe-civil

Reply:
This is a static class and the variables are considered instance variables and therefore can only be used by the each instance with its own values therefore it is thread safe.

Reply:
I don't think this is true. @UtilityClass makes all the 'instance' variables static (it isn't possible to access an instance variable from a static context)